docs/gotchas.rst

```diff
@@ -232,6 +232,18 @@ Here is an example of that:
     >>> crs_epsg == crs_deprecated
     False
 
+
+SQLite error on SELECT
+----------------------
+
+The PROJ database is based on the EPSG database. With each release,
+there is a good chance that there are database updates. If you have multiple
+versions of PROJ installed on your systems and the search path for
+the data directory becomes mixed up, you may see an error message like:
+`SQLite error on SELECT`. This is likely due to a version of PROJ
+attempting to use an incompatible database.
+
+
```

Comment:
It might be worth mentioning `pyproj.datadir.set_datadir` here, as one way to deal with this to manually set the correct path (apart from cleaning up your environment)

Reply:
Good thinking - added a section for debugging tips.